.jshintrc

```diff
@@ -3,6 +3,7 @@
   "camelcase": true,
   "curly": true,
   "eqeqeq": true,
+  "esversion": 6,
```

Comment:
Please remove this for now. Let's talk about moderninzing the library in your pull request.